README.md

```diff
@@ -39,5 +41,49 @@ Currently only `to` or `from` can be used separately. They cannot be used togeth
 #### Styling
 Styling is currently limited, but can be expanded in the future.
 
+#### `<calendar-picker />` Creation
+Vue calendar picker is easily created, and has many properties that can be changed.
+
+```HTML
+<calendar-picker v-model='dates' />
+```
+
+### `<calendar-picker />` Properties
+| Property | Required | Type | Default | Description |
+|--|--|--|--|--|
+| value | true | Object | -- | passed in the form of either `{ day: null }` for single date picking, or `{ start: null, end: null }` for multi day picking |
+| month | false | Number | `new Date().getMonth() + 1` | _**ones based**_ month to be displayed (Jan = 1, Dec = 12) |
+| year | false | Number | `new Date().getFullYear()` | year to be displayed |
+| multiDaySelect | false | Boolean | True | Allow either single, or multi day selection |
+| next | false | String | `Next` | string to be displayed for next month button |
+| prev | false | String | `Prev` | string to be displayed for previous month button |
+| highlighted | false | Object | `{}` | days that should be highlighted on load. can include tooltip information to be displayed on hover. should be passed in the format `{ [dd-MM-yyyy]: { name: 'My Tooltip Info' }}`. for example:
+```js
+{
+    "28-09-2018": {
+        name: "last friday in september"
+    },
+    "01-10-2018": {
+        name: "first monday in october"
+    }
+}
+```
+
+### Misc
+Instead of providing a `value` prop, `v-model` can be used to make use of two way data binding. Similarly, the `month` and `year` props can be bound using the `.sync` modifier.
+```html
+<!-- sync example -->
+<calendar-picker
+    v-model='selectedDates'
+    :month.sync='month'
+    :year.sync='year' />
+```
+
+
+### Styling
+All elements are namespaced starting with `.cal` and are written at the root level for the easiest customization. The calendar will grow/shrink width in relation to the surrounding div, and the squares height can be easily overwritten using `.cal-day { height: 50px; }`. The most common styles to be overridden are likely to be the background colors of: `.cal-header, .cal-inactive, .cal-today, .cal-highlighted, .cal-clicked, .cal-faded`.
+
+![calendar-picker component](https://raw.githubusercontent.com/reed-jones/vue2-date-selector/calendar-picker/assets/calendar-picker.png)
```

Comment:
Should this not be pointing to the core repo?

Reply:
probably, but the core repo doesn't have this image until this gets merged in and i didn't want to merge a broken path by accident

Reply:
Well it won't be a broken path because the file and the changes go in at the same time.